### PR TITLE
fix(flow): update docstring in flow

### LINKED ISSUE #481: fix docstring in jina.flow
**Describe the feature**

fix docstring in jina.flow

**Your proposal**

fix problems in
https://docs.jina.ai/api/jina.flow.html?highlight=flow#jina.flow.Flow

---

**Environment**

**Screenshots**

### PR DESCRIPTION
**Pull request by YueLiu-jina**
fix(flow): update docstring in flow

resolves #481

### CODE REVIEW

**Comment by hanxiao** [comment]
recheckcla

**Comment by YueLiu-jina** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by nan-wang** [comment]
resolves #481

File: jina/flow/__init__.py [hunk 1]
```diff
@@ -604,7 +603,7 @@ def index_files(self, patterns: Union[str, List[str]], recursive: bool = True,
                     **kwargs):
         """ Use a set of files as the index source for indexing on the current flow
 
-        :param patterns: The pattern may contain simple shell-style wildcards, e.g. '*.py', '[*.zip, *.gz]'
+        :param patterns: The pattern may contain simple shell-style wildcards, e.g. '\*.py', '[\*.zip, \*.gz]'
```
Comment (hanxiao) [inline jina/flow/__init__.py:606]: what is `\`??

Make sure to first read sphinx docs before you change any formatting

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html
https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

Comment (nan-wang) [reply jina/flow/__init__.py:606]: This is to escape the `*`. In the online doc, `*` is interpreted literally. 

[https://docs.jina.ai/api/jina.flow.html?highlight=flow#jina.flow.Flow.index_files](https://docs.jina.ai/api/jina.flow.html?highlight=flow#jina.flow.Flow.index_files)

**Pull request merged**